docs/guides/getting-started/getting-started-scaling-and-tuning.adoc

```diff
@@ -0,0 +1,62 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Scaling"
+summary="Get started with {project_name} scaling and tuning">
+
+Once you have {project_name} started you can next consider adapting your instance to whatever load may be needed. Scaling and tuning can help you:
+
+- minimize resource utilization
+- achieve target response times
+- minimize database pool contention
+- resolve out of memory errors, or excessive garbage collection overhead
+- provide higher availability via horizontal scaling
+
+== Vertical Scaling
+
+As you monitor your {project_name} workload if you see that CPU or memory is either under or over utilized, please consult <@links.ha id="concepts-memory-and-cpu-sizing" /> to better tune the resources available to the Java Virtual Machine (JVM).
+
+Before increasing the amount of memory available to the JVM, in particular when experiencing an out of memory error, it is best to determine what is contributing to the increased footprint via a heap dump. Excessive response times may also indicate the HTTP work queue is too large and tuning for load shedding would be better than simply providing more memory - see below.
+
+=== Common Tuning Options
+
+{project_name} automatically adjusts the number of used threads based upon how many cores you make available. Manually changing the thread count can improve overall throughput see <@links.ha id="concepts-threads" /> - but this must be done in conjunction with other JVM resources, such as database connections - see <@links.ha id="concepts-database-connections" /> as otherwise you may be moving a bottleneck somewhere else.
```

Comment:
```suggestion
{project_name} automatically adjusts the number of used threads based upon how many cores you make available. Manually changing the thread count can improve overall throughput. For more details,  see <@links.ha id="concepts-threads" />. However, changing the thread count must be done in conjunction with other JVM resources, such as database connections;  otherwise, you may be moving a bottleneck somewhere else. For more details, see <@links.ha id="concepts-database-connections" />.
```